app/models/version_status.rb

```diff
@@ -5,13 +5,13 @@
 class VersionStatus
   # Nil status for when there is no status yet since H3 is still depositing.
   class NilStatus
-    def status_message
-      'Saving'
-    end
+    def status_message = 'Saving'
 
-    def draft?
-      false
-    end
+    def draft? = false
```

Comment:
Wow, never seen this syntax before.

Reply:
@justinlittman Yeah, it's new as of Ruby 3.x.